go/test/integration/BUILD

```diff
@@ -0,0 +1,5 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
+
+exports_files(
```

Comment:
Need to put a `TODO` on top of the test target to move it here. Also, add a `TODO` to highlight that the test doesn't build because of the linking issues (while `go_binary` is fine).

Reply:
This seems to not have been addressed?

Reply:
I believe that we have a pretty specific todo in the main BUILD file, so it's enough. Thanks for being attentive though